app/workers/evss/document_upload.rb

```diff
@@ -12,3 +12,13 @@ def perform(auth_headers, user_uuid, document_hash)
     uploader.remove!
   end
 end
+
+# Allows gracefully migrating tasks in queue
+# TODO(knkski): Remove after migration
+class EVSSClaim::DocumentUpload
+  include Sidekiq::Worker
+
+  def perform(auth_headers, user_uuid, document_hash)
+    EVSS::DocumentUpload.new.perform(auth_headers, user_uuid, document_hash)
```

Comment:
Let's change this to a `perform_async` so it removes it from the `EVSSClaim::DocumentUpload` queue and adds it as an `EVSS::DocumentUpload` job